ReactiveFeedback.xcodeproj/xcshareddata/xcschemes/Example.xcscheme

```diff
@@ -80,7 +80,7 @@
             buildForTesting = "YES"
             buildForRunning = "YES"
             buildForProfiling = "YES"
-            buildForArchiving = "YES"
+            buildForArchiving = "NO"
```

Comment:
Was this change intentional?

Reply:
Yes, otherwise this: https://github.com/SwipeCellKit/SwipeCellKit/issues/297

Reply:
This is for the Example target, so it wouldn't impact RAF consumers.